roles/container-storage-setup/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- block:
+    - name: create the docker-storage config file
+      template:
+        src: docker-storage-setup-overlayfs.j2
+        dest: /etc/sysconfig/docker-storage-setup
+        owner: root
+        group: root
+        mode: 0644
+  when:
+    - openshift_openstack_ephemeral_volumes == false 
```

Comment:
As this role will also be called for non-openstack IaaS, we should make this variable a bit more generic - maybe something like `openshift_configure_container_storage` (then in one of the `openstack` specific playbooks - perhaps `playbooks/openshift/openstack/post-provision.yml` - set the fact correctly based on the `openshift_openstack_ephemeral_volumes` value).